pom.xml

```diff
@@ -339,6 +339,11 @@
 				<artifactId>archaius-core</artifactId>
 				<version>${archaius.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>${gs-test.groupId}</groupId>
+				<artifactId>gs-test</artifactId>
+				<version>14.0.7</version>
```

Comment:
Please note that this version of `gs-test` must be built before this MR can be merged.

Reply:
`gs-test:14.0.7` is now built.